test/integration/narrow/routes/agents-details.test.js

```diff
@@ -1,51 +1,54 @@
 const { crumbToken } = require('./test-helper')
-
 describe('Agent details page', () => {
   const varList = {
     applicant: 'Farmer',
     applying: 'Agent'
   }
-
-  jest.mock('../../../../app/helpers/session', () => ({
+  jest.mock('../../../../app/helpers/functions/session', () => ({
     setYarValue: (request, key, value) => null,
     getYarValue: (request, key) => {
-      console.log(key, 'key')
       if (varList[key]) return varList[key]
       else return 'Error'
     }
   }))
-
   it('should load page successfully', async () => {
     const options = {
       method: 'GET',
       url: `${global.__URLPREFIX__}/agents-details`
     }
-
     const response = await global.__SERVER__.inject(options)
     expect(response.statusCode).toBe(200)
   })
-
   it('should return various error messages if no data is entered', async () => {
     const postOptions = {
       method: 'POST',
       url: `${global.__URLPREFIX__}/agents-details`,
       payload: { crumb: crumbToken },
       headers: { cookie: 'crumb=' + crumbToken }
     }
-
     const postResponse = await global.__SERVER__.inject(postOptions)
     expect(postResponse.statusCode).toBe(200)
-    expect(postResponse.payload).toContain('Enter your first name')
-    expect(postResponse.payload).toContain('Enter your last name')
-    expect(postResponse.payload).toContain('Enter your business name')
-    expect(postResponse.payload).toContain('Enter your email address')
-    expect(postResponse.payload).toContain('Enter a landline number (if you do not have a landline, enter your mobile number)')
-    expect(postResponse.payload).toContain('Enter your address line 1')
-    expect(postResponse.payload).toContain('Enter your town')
-    expect(postResponse.payload).toContain('Select your county')
-    expect(postResponse.payload).toContain('Enter your postcode, like AA1 1AA')
+    const htmlPage = createPage(postResponse.payload)
```

Comment:
Given the test before these changes checked which error messages appeared with minimal coding, is it necessary to refactor this?